### ISSUE TITLE
Conditional assignment operator throwing error

### ISSUE THREAD

**Issue by PalleZingmark** [rossPatton/stylint#215]
I've used the same setup as used here: https://github.com/rossPatton/stylint/issues/212#issuecomment-151407484 

**test.styl:**
```css
$foo-color ?= #0976b5;
```

>Warning: always use brackets when defining selectors
>File: test.styl
>Line: 1: $foo-color ?= #0976b5;

**Comment by Cordazar** [comment]
The same goes for 
```
$x += $i;
$x -= $i;
```

**Comment by rossPatton** [comment]
this should be fixed as of 1.3.0
